src/tool_fsharp/tool_fsc.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.tasks" />
+
+  <PropertyGroup>
+    <VersionPrefix>$(CliVersionPrefix)</VersionPrefix>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PublishDir>$(FSharpDirectory)</PublishDir>
+    <VersionSuffix>$(CommitCount)</VersionSuffix>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NetCore.App" Version="$(CLI_SharedFrameworkVersion)" />
+    <PackageReference Include="Microsoft.FSharp.Core.netcore" Version="$(CLI_FSharp_Version)" />
+    <PackageReference Include="Microsoft.FSharp.Compiler.netcore" Version="$(CLI_FSharp_Version)" />
```

Comment:
I spoke with @KevinRansom in person.  The code for this stuff in https://github.com/Microsoft/VisualFSharp as you saw, @omajid.  The plan is Kevin is going to get these nuget packages building using a pre-release version of the 2.0 CLI (once his changes are merged and it can build fsproj's) and then share out how to build them, which we'll include in the Build From Source effort.  By using the SDK like we do in all the other projects, we can remove the need to take a dependency on mono to build the CLI.  Does that seem reasonable?